### PR TITLE
display error icon on non-success

### PR DESCRIPTION
**Pull request by ArtificialOwl**
Signed-off-by: Maxence Lange <maxence@nextcloud.com>

### CODE REVIEW

**Comment by daita** [comment]
https://gfycat.com/ChubbyMadHyrax

File: settings/css/settings.css [hunk 1]
```diff
@@ -175,6 +175,13 @@ input#openid, input#webdav {
 	pointer-events: none;
 }
 
+#personal-settings-container > div > form span[class^="icon-error"] {
```
Comment (pixelipo) [inline settings/css/settings.css]: why such a complicated selector? Wouldn't `#personal-settings-container .icon-error` suffice?

Or, you could just modify existing selector on line 167 instead...

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Comment by MorrisJobke** [comment]
@daita Could you create the master PR please? Thanks

**Review by pixelipo — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Let me forward port this.

**Comment by daita** [comment]
my bad, totally forgot about it

**Comment by MorrisJobke** [comment]
> my bad, totally forgot about it

Don't worry :)

**Comment by MorrisJobke** [comment]
Regression is fixed via #6098

**Pull request merged**